src/pcrlock/pcrlock.c

```diff
@@ -2114,6 +2114,9 @@ static int show_log_table(EventLog *el, JsonVariant **ret_variant) {
         FOREACH_ARRAY(rr, el->records, el->n_records) {
                 EventLogRecord *record = *rr;
 
+                if (arg_pcr_mask && !FLAGS_SET(arg_pcr_mask, UINT32_C(1) << record->pcr))
```

Comment:
nitpick, as commented elsewhere, for non-boolean and non-pointer values pleas do explicit numeric comparisons. i.e. arg_pcr_mask is not a boolean or pointer, hence please check it for non-zero with an explicit `!= 0`